.github/workflows/base-docker-image.yml

```diff
@@ -0,0 +1,22 @@
+name: "Generate apply-base docker image"
+on:
+  schedule:
+    - cron:  '45 5 * * *'
+  workflow_dispatch:
+jobs:
+  build-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PAT }}
```

Comment:
have you changed the vars here yet?

Reply:
Yes, both Circle and GA have had the credentials ipdated :+1: